apps/testnorge-hodejegeren/hodejegeren-core/src/test/java/no/nav/registre/hodejegeren/provider/rs/CacheControllerIntegrationTest.java

```diff
@@ -0,0 +1,27 @@
+package no.nav.registre.hodejegeren.provider.rs;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureWireMock(port = 0)
+@AutoConfigureMockMvc
+@DirtiesContext
+@TestPropertySource(locations = "classpath:application-test.properties")
+class CacheControllerIntegrationTest {
```

Comment:
Se kommentarer for organisasjonControllerIntegrationTest. Kan også fjerne de andre tomme integrasjon testene i hodejegeren sånn at det ikke blir for mange tomme tester (HistorikkControllerIntegrationTest og HodejegerenControllerIntegrationTest)